djmoney/contrib/exchange/backends/base.py

```diff
@@ -19,8 +20,8 @@
 
 
 class BaseExchangeBackend:
-    name = None
-    url = None
+    name: Optional[str] = None
+    url: Optional[str] = None
```

Comment:
Okay I'm not sure about this, but according to Adam J's latest blog post, this might be old syntax? https://adamj.eu/tech/2022/10/18/python-type-hints-implicit-optional-types/

Reply:
I believe, this syntax was introduced in Python 3.10, so likely we need to support the old one for a while